packages/nimble-components/src/table/specs/table-column-specs/table-column-text-field.md

```diff
@@ -0,0 +1,185 @@
+# TableColumnText
+
+## Overview
+
+The `nimble-table-column-text` is a component that defines how to render a cell for that column in a `nimble-table` as text.
+
+### Background
+
+[Table Column API](../table-columns-hld.md)
+
+[Table Spec](../README.md)
+
+[Visual Design Spec](https://xd.adobe.com/view/5b476816-dad1-4671-b20a-efe796631c72-0e14/specs/)
+
+### Non-goals
+
+-   Defining an API that supports editable text. Once we need editable text we will determine if it makes sense to modify this component or introduce a new column type.
+
+### Features
+
+-   Offers a `placeholder` attribute that allows a user to define what text to render when no value is provided by the `nimble-table` data.
+
+---
+
+## Design
+
+Below is an example of how the `nimble-table-column-text` would be used within a `nimble-table`:
+
+```HTML
+<nimble-table>
+    <nimble-table-column-text value-key="firstName" placeholder="No data">First Name</nimble-table-column-text-field>
+    <nimble-table-column-text value-key="lastName" placeholder="No data">Last Name</nimble-table-column-text-field>
+</nimble-table>
+```
+
+### API
+
+_Component Name_
+
+-   `nimble-table-column-text`
+
+_*Props/Attrs*_
+
+-   `value-key`: string
+-   `placeholder`: string
+
+_Type Reference_
+
+-   [`TableColumn`](../table-columns-hld.md#tablecolumn)
+-   [`StringField`](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/table/specs/table-data-api.md#implementation--design) (section showing example types)
+-   [`TableCellState`](../table-columns-hld.md#tablecellstate-interface)
+
+The `TableColumnText` will extend the `TableColumn` in a manner similar to the following:
+
+```TS
+type TableColumnTextCellData = StringField<'value'>;
+type TableColumnTextColumnConfig = { placeholder: string };
+
+public class TableColumnText extends TableColumn<TableColumnTextCellData, TableColumnTextColumnConfig> {
+    ...
+
+    @attr({ attribute: 'value-key'})
+    public valueKey: string;
+
+    @attr
+    public placeholder: string; // Column auxiliary configuration
+
```

Comment:
We also need to implement [cellStateDataFieldNames](https://github.com/ni/nimble/blob/b7aa70e5153e9f5dcdd1b9a2d671000accdc8b7a/packages/nimble-components/src/table/specs/table-columns-hld.md#tablecolumn) and looks like the chose name is 'value' which seems fine

Reply:
Updated.